src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
[Optional[
- Kindly remove disabling rules

Reply:
Yeah, I will do that, and Thanks for the review. I think it was showing that error because I was using class-based components but I will resolve this on the next milestone.
 thanks again.